docs/UserGuide.md

```diff
@@ -67,13 +67,90 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
 
 Shows a message listing all the main commands and explaining how to access the user guide.
 
-Precise command format: `help`
+#### Format: `help`
 
-Precise expected outcome on success:
-A pop-up of a new window that lists out all the main commands with their respective details and displays a link to FAPro’s user guide.
+![help format](images/helpFormat.png)
+
+#### Precise expected outcome on success:
+* Successful message: Opened help window
+* A pop-up window lists all the main commands and displays a link to FAPro's user guide
 
 ![help window](images/helpWindow.png)
 
+### Viewing risk assessment questionnaire: `questionnaire`
+
+Displays questions that are going to be used for generating client risk profile levels.
+
+#### Format: `questionnaire`
+
+![questionnaire format](images/questionnaireFormat.png)
+
+#### Precise expected outcome on success:
+* Successful message: Opened questionnaire window
+* A pop-up window shows the risk assessment multiple choice questions, including the grading and risk profile categories criteria
+
+![questionnaire window](images/questionnaireWindow.png)
+
+### Adds risk profile level to a contact:`riskprofile`
+
+Generates the client risk profile level based on their response of risk assessment questionnaire provided in Questionnaire tab and adds it to their contact.
```

Comment:
Also, just another formatting issue. We are removing full stops :D Minor issue!

Reply:
Fixed!